config/config.go

```diff
@@ -139,13 +138,12 @@ type Authorization struct {
 func New(path string) (*Config, error) {
 	f, err := os.OpenFile(path, os.O_CREATE|os.O_RDONLY, 0600)
 	if err != nil {
-		return nil, errors.Wrap(err, "OpenFile failed")
+		return nil, errors.Wrap(err, "config read failed")
```

Comment:
I think os.OpenFile is not read function, just open file desctiptor only,
so, `os.OpenFile` or `open file` is better than `config read`

Reply:
I am thinking line 149,
```go
f, err := os.OpenFile(path, os.O_CREATE|os.O_RDONLY, 0600)
```
maybe it is better to remove os.O_CREATE flag while open the file??

Reply:
Opps,

I think so, can you remove it?

Reply:
Sure :)